src/admin/collectionsOrBundles/helpers/collection-filters.ts

```diff
@@ -95,15 +95,27 @@ export function generateCollectionWhereObject(
 	}
 	if (filters.education_levels && filters.education_levels.length) {
 		andFilters.push({
-			_or: [
-				...getMultiOptionsFilters(
-					{
-						education_levels: without(filters.education_levels, NULL_FILTER),
-					},
-					['education_levels'],
-					['lom_context']
-				),
-			],
+			...getMultiOptionsFilters(
+				{
+					education_levels: without(filters.education_levels, NULL_FILTER),
+				},
+				['education_levels'],
+				['lom_context']
+			)[0],
```

Comment:
there will only be one entry in the array, so you can just omit the [0]. 
since push accepts multiple entries via:
.push(...someArray);

also update the `filters.subjects` to match this notation since the subjects now still include a redundant _or statement:
```json
  {
    "_or": [
      {
        "lom_classification": {
          "_has_keys_any": [
            "aardrijkskunde",
            "autotechniek"
          ]
        }
      }
    ]
  },
```

Reply:
fixed